Source/WebKit/UIProcess/API/Cocoa/WKPreferencesInternal.h

```diff
@@ -42,3 +44,13 @@ template<> struct WrapperTraits<WebPreferences> {
 }
 
 @end
+
+#endif
+
+@interface WKPreferences ()
+
+#if TARGET_OS_IPHONE
```

Comment:
Nit - we prefer using `PLATFORM()` when possible.

```suggestion
#if PLATFORM(IOS_FAMILY)
```

Reply:
ah, I forgot we can do that in Internal headers!